Assets/Scripts/Inventory/Chest.cs

```diff
@@ -0,0 +1,170 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Chest : PlacedObject
+{
+    public List<Item> contents = new List<Item>();
+
+    public GlobalStorage globalStorage;
+
+    public Vector3 coordinate;
+
+    /// Basic constructor for chest - sets it up so that it hooks up w/ GlobalStorage system
+    public Chest(GlobalStorage gs)
+    {
+        globalStorage = gs;
+        globalStorage.inventories.Add(this);
+    }
+
+    /// Constructor for chest - sets it up so that it hooks up w/ GlobalStorage system
+    /// Also sets the coordinate of the chest
+    public Chest(GlobalStorage gs, Vector3 coord)
+    {
+        globalStorage = gs;
+        globalStorage.inventories.Add(this);
+        coordinate = coord;
+    }
+
+    /// <summary>
+    /// <para>Checks if the chest contains the specified item</para>
+    /// <para>Returns true if it does, false if it doesn't</para>
+    /// </summary>
+    public bool DoesContainItem(Item item)
+    {
+        bool contains = contents.Contains(item);
```

Comment:
We may have to start moving away from comparing Item objects to comparing Item objects' 'Name' variable to get a reference to a specific object in an inventory. Will mention it at this upcoming weekly meeting and this is fine for now.